values.yaml

```diff
@@ -423,6 +423,8 @@ core:
     tag: dev
   # set the service account to be used, default if left empty
   serviceAccountName: ""
+  # Use this parameter to set the schedulername for this element. Useful to use Portworx with Harbor
+  schedulerName: ""
```

Comment:
How about introducing a global configuration item for the scheduler rather than configuring for all of the pods?

Reply:
Agreed. New Pull rewquest created with the new changes to the code. This time the values file only has one new parameter to set the schedulerName globally. Every template reads that parameter.